include/core/parameters_lagrangian.h

```diff
@@ -250,15 +298,16 @@ namespace Parameters
     class BoundaryMotion
     {
     public:
-      // Number of moving boundaries
-      unsigned int moving_boundary_number;
 
-      // Choosing motion type
-      enum class motion_type
+      // Choosing motion method
+      enum class MotionMethod
       {
-        translational,
-        rotational
-      } motion_type;
+        free,
+        fix
+      } motion_method;
+
+      // Number of moving boundaries
+      unsigned int moving_boundary_number;
```

Comment:
I would keep the parameter related to the dynamic motion of the cylinder out of this PR, but also into another set of parameter. I would like the cylinder calibration problem to be as self-contained as possible.

Reply:
Will try that